articles/libraries/lock/v9/index.md

```diff
@@ -23,6 +23,10 @@ Lock makes it easy to integrate SSO in your app. You won't have to worry about:
 ![Lock Sign Up](/media/articles/libraries/lock/v9/lock-signup.png)
 
 ::: note
+Check out the [Lock repository](https://github.com/auth0/lock) on GitHub.
+:::
+
+::: note
```

Comment:
Same here, change this so it's not a 'double note'?

Reply:
No, leave it as is since the topics are not related

Reply:
I think that you should link to this repo https://github.com/auth0/lock/tree/v9 

cc @jeffreylees